src/elements/content-sidebar/versions/__tests__/VersionsSidebarContainer.test.js

```diff
@@ -191,10 +191,26 @@ describe('elements/content-sidebar/versions/VersionsSidebarContainer', () => {
         test('should set state to default values with error message', () => {
             const wrapper = getWrapper();
 
-            wrapper.instance().handleFetchError();
+            wrapper.instance().handleFetchError({ status: 500 });
 
             expect(wrapper.state()).toEqual({
```

Comment:
Optional, but we can change `toEqual` to `toMatchObject` and only check the props we care about to reduce the surface area of these tests.

Reply:
I sure can!